.github/workflows/layers_govcloud.yml

```diff
@@ -101,12 +101,16 @@ jobs:
       - name: Create Layer
         id: create-layer
         run: |
-          LAYER_VERSION=$(aws --region us-gov-east-1 lambda publish-layer-version \
-            --layer-name AWSLambdaPowertoolsTypeScriptV2 \
+          cat AWSLambdaPowertoolsTypeScriptV2.json | jq '{
```

Comment:
Just to double check - do we need to add `-r` to the `jq` command?

If not, ignore and resolve the convo.

Reply:
It didn't care when I tested locally, the command creates this: 

```json
{
  "LayerName": "AWSLambdaPowertoolsTypeScriptV2",
  "Description": "Powertools for AWS Lambda (TypeScript) version 2.0.2",
  "CompatibleRuntimes": [
    "nodejs16.x",
    "nodejs18.x",
    "nodejs20.x"
  ],
  "LicenseInfo": "MIT-0"
}
```